### PR TITLE
OD-17689 submodule xapi-proto add gpid to ImpExt_rp

### PR DESCRIPTION
**Pull request by BenjaminWassermann**
https://magnite.atlassian.net/browse/OD-17689

### CODE REVIEW

File: src/proto/com/magnite/openrtb/v2/openrtb-xapi.proto [hunk 1]
```diff
@@ -138,6 +138,13 @@ message ImpExt {
 
         // Floor data used to synthesize RTB rules.
         repeated Floor floors = 9;
+
+        // OD-17689 GPID needs to be handled by ImpExt_rp
+        // Global Placement ID. The unique identifier of the placement that is sent 
+        // to all participants in buying impressions for this placement. In general, 
+        // this field will be populated with one of the DFP Ad Unit Code, Prebid Ad Slot,
+        // or value provided by the publisher or ad server.
```
Comment (eskenny) [inline src/proto/com/magnite/openrtb/v2/openrtb-xapi.proto:146]: This phrasing doesn't really make sense to me. It seems like it's talking about us populating it? This is for the incoming bid request, not something we have any control over—the value would always be populated by the publisher/ad server.

Comment (eskenny) [reply src/proto/com/magnite/openrtb/v2/openrtb-xapi.proto:146]: Ah, didn't realize that. I guess that's ok then. I'd still suggest maybe changing in both comments "will" to "should" and "value" to "some value." The original PR (https://github.com/MagniteEngineering/xapi-proto/pull/4) has more context on the intention.

File: src/proto/com/magnite/openrtb/v2/openrtb-xapi.proto [hunk 2]
```diff
@@ -138,6 +138,13 @@ message ImpExt {
 
         // Floor data used to synthesize RTB rules.
         repeated Floor floors = 9;
+
+        // OD-17689 GPID needs to be handled by ImpExt_rp
```
[the remark this answers is not in the document]
Comment (BenjaminWassermann) [reply src/proto/com/magnite/openrtb/v2/openrtb-xapi.proto]: Done

File: src/proto/com/magnite/openrtb/v2/openrtb-xapi.proto (hunk 1 above)
Comment (BenjaminWassermann) [reply src/proto/com/magnite/openrtb/v2/openrtb-xapi.proto]: done

**Review by eskenny — APPROVED** [review]

**Pull request merged**